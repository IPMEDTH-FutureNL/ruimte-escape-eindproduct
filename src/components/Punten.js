import React, { Component } from 'react'

export class Punten extends Component {
    constructor(props){
        super(props);
    }
  render() {
    return (
      <div className="punten">Punten: 100</div>
    )
  }
}

export default Punten