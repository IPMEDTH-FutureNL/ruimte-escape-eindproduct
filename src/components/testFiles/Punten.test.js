import renderer from 'react-test-renderer';
import Punten from "../Punten.js";

it('Test if point are changed when necessary', () => {
    const component = renderer.create(
        <Punten punten="50"></Punten>,
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
})