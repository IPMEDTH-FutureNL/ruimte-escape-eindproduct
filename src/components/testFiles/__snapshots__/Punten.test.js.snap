// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test if point are changed when necessary 1`] = `
<div
  className="punten"
>
  Punten: 
  50
</div>
`;
